src/Lesson8/Task5.java

```diff
@@ -0,0 +1,18 @@
+package Lesson8;
+/*
+Throw a NullPointerException from the method iCanThrowException();
+Handle it in the main method.
+ */
+public class Task5 {
+    public static void main(String[] args) {
+        try {
+            iCanThrowException();
+        } catch (Exception e) {
```

Comment:
iCanThrowException returns NullPointerException exception. Why do you handle Exception instead of NullPointerException?